src/main/java/ay2021s1_cs2103_w16_3/finesse/ui/UiState.java

```diff
@@ -10,10 +10,31 @@
      * Tabs present in the UI.
      */
     public enum Tab {
-        OVERVIEW,
-        INCOME,
-        EXPENSES,
-        ANALYTICS;
+        OVERVIEW(1),
+        INCOME(2),
+        EXPENSES(3),
+        ANALYTICS(4);
+
+        /** The index of the tab in the {@code TabPane}. */
+        private final int tabIndex;
+
+        /**
+         * Constructs a new {@code Tab} enum with the specified tab index.
+         *
+         * @param tabIndex The index of the tab in the {@code TabPane}.
+         */
+        private Tab(int tabIndex) {
```

Comment:
![image](https://user-images.githubusercontent.com/53135010/95965916-17cc7000-0e3d-11eb-86de-02d5c0bc373b.png)

Does this need to be fixed?